Specs/DateTime/German/DateTimePeriodParser.json

```diff
@@ -259,40 +285,40 @@
     ]
   },
   {
-    "Input": "Ich gehe diesen Abend zurück",
+    "Input": "Ich komme heute Abend zurück",
     "Context": {
       "ReferenceDateTime": "2016-11-07T16:12:00"
     },
-    "NotSupported": "javascript, python",
+    "NotSupported": "javascript, python, dotNet",
     "Results": [
       {
-        "Text": "diesen Abend",
+        "Text": "heute Abend",
         "Type": "datetimerange",
         "Value": {
           "Timex": "2016-11-07TEV",
           "FutureResolution": {
-            "startDateTime": "2016-11-07 16:00:00",
-            "endDateTime": "2016-11-07 20:00:00"
+            "startDateTime": "2016-11-07 18:00:00",
+            "endDateTime": "2016-11-07 22:00:00"
           },
           "PastResolution": {
-            "startDateTime": "2016-11-07 16:00:00",
-            "endDateTime": "2016-11-07 20:00:00"
+            "startDateTime": "2016-11-07 18:00:00",
+            "endDateTime": "2016-11-07 22:00:00"
           }
         },
-        "Start": 9,
-        "Length": 12
+        "Start": 10,
+        "Length": 11
       }
     ]
   },
   {
-    "Input": "An diesem Morgen",
+    "Input": "An diesem Vormittag",
```

Comment:
Same

Reply:
Someone translated the English spec "I'll go back this morning" to just "An diesem Morgen" rather than "Ich komme an diesem Morgen zurück". And "An diesem Morgen" or "An diesem Vormittag" would both be translated to "On this Morning" in English, but mean different things in German. I will just add several variations.